tests/factories/RemoteComment.js

```diff
@@ -0,0 +1,10 @@
+import { define } from "factoria";
+
+export default define( "RemoteComment", faker => ( {
+  id: faker.datatype.number( ),
+  uuid: faker.datatype.uuid( ),
+  created_at: "2021-11-09T08:28:05-08:00",
+  updated_at: "2021-11-09T17:22:17-08:00",
+  user: factory( "RemoteUser" ),
```

Comment:
Noticed this one is missing a `factory` import. Should we set up some [pre-commit hooks](https://markoskon.com/create-a-pre-commit-hook-with-husky-and-lint-staged/) to run `eslint` (and maybe `jest`) before every commit?

Reply:
Will fix. Def. in favor of a pre-commit eslint. Pre-commit jest is overkill, IMO, because running all the tests takes a while, and we're going to be doing that anyway with Github Actions.